src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -6222,6 +6222,22 @@ def test_network_vnet_gateway_edge_zone(self, resource_group):
                  '--edge-zone {edge_name}',
                  checks=self.check('vnetGateway.extendedLocation.name', '{edge_name}'))
 
+    @ResourceGroupPreparer(name_prefix='test_network_vnet_gateway_local_gateway', location='eastus2euap')
+    def test_network_vnet_gateway_local_gateway(self, resource_group):
+
+        self.kwargs.update({
+            'rg': resource_group,
+            'vnet': 'vnet',
+            'gateway_type' : 'LocalGateway',
+            'edge_name': 'microsoftrrdclab3',
+            'edge_zone_vnet_id' : 'edge_zone_vnet_id'
+        })
```

Comment:
```suggestion
        self.kwargs.update({
            'rg': resource_group,
            'vnet': 'vnet',
            'gateway_type' : 'LocalGateway',
            'edge_name': 'microsoftrrdclab3',
        })
```
It's useless as we declare it later.